abuild.in

```diff
@@ -61,6 +61,32 @@ error() {
 	logcmd "ERROR: $pkgname: $1"
 }
 
+amove() {
+	[ -n "${1##/*}"   ] || return 1
+	[ -n "$subpkgdir" ] || return 1
+	[ -e "$pkgdir/$1" ] || return 1
+
+	# store directory
+	d="$(pwd -L)"
```

Comment:
can't we use OLDPWD ?

Reply:
There are multiple instances of `cd` throughout the script, so `OLDPWD` is not guaranteed to be the value we expect it to be.